js/index.js

```diff
@@ -40,3 +41,93 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.getElementById('cta-img');
+ctaImg.setAttribute('src', siteContent['cta']['img-src']);
+
+let middleImg = document.getElementById('middle-img');
+middleImg.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+let nav = document.getElementsByTagName('nav')[0];
+// create new anchor to append to the nav bar
+let newNavLink = document.createElement('a');
+nav.appendChild(newNavLink);
+
+
+// Update the header children
+let navLinks = document.getElementsByTagName('a');
+for(let i = 0; i < navLinks.length; i++) {
+  let navItem = navLinks.item(i);
+  let innerText = 'nav-item-' + (i + 1);
```

Comment:
Good use of a for loop here. You could change it to be a `forEach()` if you wanted to.